modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryTableStorage.java

```diff
@@ -364,7 +364,10 @@ private GcQueue createGcQueue(
     CompletableFuture<Void> destroyMvPartitionStorage(AbstractPageMemoryMvPartitionStorage mvPartitionStorage) {
         // It is enough for us to close the partition storage and its indexes (do not destroy). Prepare the data region, checkpointer, and
         // compactor to remove the partition, and then simply delete the partition file and its delta files.
-        mvPartitionStorage.close();
+        boolean transitioned = mvPartitionStorage.transitionToDestroyedState();
```

Comment:
Variable is not needed right?

Reply:
It is introduced just to make the code more readable, it explains what it means. If I inline it, the code would look like `if mvPartitionStorage.transitionToDestroyedState()`, and it's not obvious right away what would `true` or `false` mean in this case. But if there is a variable called `transitioned`, then it's obvious that the transition we tried was successful.

Reply:
For me, the variable does not add more clarity, at your discretion, of course, I would remove it, since the code is not complicated and understandable.